src/components/SortBar.jsx

```diff
@@ -0,0 +1,55 @@
+import { useState, useEffect } from "react"
+import { getTopics } from "../api"
+
+const SortBar = ({sortBy, setSortBy, order, setOrder}) => {
+    const [categories, setCategories] = useState([])
+    const [sortByMsg, setSortByMsg] = useState('')
+    const [orderMsg, setOrderMsg] = useState('')
+
+    useEffect(() => {
+        sortBy ? setSortByMsg(`Sorting by: ${sortBy.displayName}`): ''
+    }, [sortBy])
+
+    useEffect(() => {
+        order ? setOrderMsg(`Ordering by: ${order.displayName}`): ''
+    }, [order])
+
+    useEffect(() => {
+        getTopics()
+        .then((fetchedCategories) => {
+            setCategories([{
+                slug: 'All',
+                description: 'All Categories'
+            }, ...fetchedCategories])
+        })
+    }, [])
+
+    return <div>
+        <h2>Sort By:</h2>
+        <p> {sortByMsg} </p>
+        <button key='date' 
```

Comment:
we're able to sort by date but we cant see the dates on the article cards so the user cant be sure that the articles have actually sorted correctly